src/Module/Magic.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file src/Module/Magic.php
+ */
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Model\Contact;
+use Friendica\Util\HTTPSig;
+use Friendica\Util\Network;
+
+use dba;
+
+/**
+ * Magic Auth (remote authentication) module.
+ * 
+ * Ported from Hubzilla: https://framagit.org/hubzilla/core/blob/master/Zotlabs/Module/Magic.php
+ */
+class Magic extends BaseModule
```

Comment:
Please rename class to `MagicAuth`.

Reply:
I have talked with @annando about the naming of the modules. If we keep it this way we are compatible with Hubzilla and maybe some day we can interact with hubzilla

Reply:
Fair enough, I didn't realize it was a module. In the future I will want to enable a module correctly named `MagicAuth` to be reachable by `/magic`.